doc/installation.md

```diff
@@ -37,18 +37,21 @@ Installation of cameras on Linux is lengthy compared to other supported platform
   * `sudo cp config/99-realsense-libusb.rules /etc/udev/rules.d/`
   * Reboot or run `sudo udevadm control --reload-rules && udevadm trigger` to enforce the new udev rules
 3. Next, choose one of the following subheadings based on desired machine configuration / kernel version (and remember to complete step 4 after). **Note: ** Multi-camera support is currently NOT supported on 3.19.xx kernels. Please update to 4.4 stable. 
-  * **Updated 4.4 Stable Kernel** (recommended)
+  * **Updated 4.4 Stable Kernel**
     * Run the following script to install necessary dependencies (GCC 4.9 compiler and openssl) and update kernel to v4.4-wily
       * `./scripts/install_dependencies-4.4.sh`
     * Run the following script to patch uvcvideo.ko
       * `./scripts/patch-uvcvideo-4.4.sh v4.4-wily` (note the argument provided to this version of the script)
       * This script involves shallow cloning the Linux source repository (~100mb), and may take a while
-  * **(OR) Stock 3.19.xx Kernel in 14.04.xx** (not recommended)
```

Comment:
I prefer to keep (not recommended) because there is an unresolved issue with mutli-cam support in 3.19